### ISSUE TITLE
Suppress RuntimeException?

### ISSUE THREAD

**Issue by francoism90** [dmitry-ivanov/laravel-console-mutex#14]
To prevent overlapping I'm using your package that basically runs a command in a loop.
This is all working fine, only the log is being flood with `RuntimeException(code: 0): Command is running now!..`. Is it possible to log a (debug) message instead of a full exception? Do you maybe have a better idea? :)

Thanks for your package!

**Comment by dmitry-ivanov** [comment]
First of all - thank you for your kind words! 🙂 

According to your situation, here is what I think:

- Maybe you could run your console command _not_ in the loop, but via [Laravel Scheduler](https://laravel.com/docs/master/scheduling#scheduling-shell-commands), for example by using `everyMinute()` method? That way, you'll still have protection from overlapping, but these situations would become rare events, which potentially require some actions.

- You can use [Laravel Console Logger](https://github.com/dmitry-ivanov/laravel-console-logger) to separate log files of your console commands from the main Laravel log file(s).

- You can handle `RuntimeException('Command is running now!')` in the `App\Exceptions\Handler` class and basically provide any custom logic you want. Something like that:

```php
if ($this->isConsoleMutexException($exception)) {
    // Any custom logic here...
}

// ...

protected function isConsoleMutexException(Exception $exception)
{
    return ($exception instanceof RuntimeException)
        && ($exception->getMessage() == 'Command is running now!');
}
```

I hope it helps! 🙂

**Comment by francoism90** [comment]
@dmitry-ivanov Many thanks! Added this in `report()` yesterday and it doesn't log anymore.

Keep up the good work.

**Comment by dmitry-ivanov** [comment]
@francoism90 Hey, hope you're doing great! 🙂 

Just wanted to let you know that I've added the [MutexRuntimeException](https://github.com/dmitry-ivanov/laravel-console-mutex/blob/df76a5ff3c91dbc0079b578ad5fd2d9b407874d7/src/MutexRuntimeException.php) class to make checks easier.

You'd be able to simplify your check to:

```php
use Illuminated\Console\MutexRuntimeException;

protected function isConsoleMutexException(Exception $exception)
{
    return $exception instanceof MutexRuntimeException;
}
```

PS: I've already pushed it to `master`, and it would be a part of the `6.0.0` release soon.
